### DISCUSSION TITLE
Longer URLs cut short in Top Pages details

### DISCUSSION THREAD

**Discussion by RobertJoonas** [plausible/analytics#1448] [Anything Else]
Currently, when I go to the Top Pages report and open up the details section, I see my longer URLs cut short. It's annoying when I always have to hover the mouse over the URLs, to see the actual, full URL.

Would be nice if I could enlarge this column to be able to see it all instantly.

(feedback via email)

Example screenshot:

![image](https://user-images.githubusercontent.com/56999674/140468683-7b82479d-5d50-4931-8c9d-a214ce69a085.png)

**Comment by metmarkosaric** [comment]
**Update:** This has now been improved for laptops and desktops. We now shorten long page paths in the middle and for extra long page paths we shorten them towards the end too. Displaying the full page path for very long cases is not something we can do at the moment without a bigger change to the way the design is.

**Reply by rajbos** [reply]
For the opening view it seems to work:

![image](https://github.com/plausible/analytics/assets/6085745/2e188ed2-237c-41ee-a8ae-0a798d4ce9f7)

But for the actual details page I still see this:
![image](https://github.com/plausible/analytics/assets/6085745/131c1350-6970-431a-ae71-e11b9e3881f4)

**Reply by rajbos** [reply]
iOS / Safari btw

**Reply by metmarkosaric** [reply]
thanks for the feedback @rajbos! I should have been more specific about what we changed. we now shorten long page paths in the middle and for extra long page paths we shorten them towards the end in addition to the middle. this makes is easier to see what page you're looking the stats for compared to our previous method of shortening the end.  but this is mostly relevant for laptops and desktops. displaying the full page path or going over several lines is not something we can do without a bigger change to the design and on mobile screens it's more difficult as well as there's way less space. we'll see if there's something else we can do to improve this especially on mobiles in the future
